pgsql/CMakeLists.txt

```diff
@@ -16,11 +20,16 @@ set ( PC_INSTALL_EXENSIONS
   "${PROJECT_BINARY_DIR}/pgsql/pointcloud.control"
   )
 
-configure_file(
-    pointcloud.sql.in
-    "${PROJECT_BINARY_DIR}/pgsql/pointcloud--${POINTCLOUD_VERSION}.sql"
+configure_file(pointcloud.sql.in pointcloud.sql)
+configure_file(li3ds.sql.in li3ds.sql)
```

Comment:
Maybe, the path in the build tree should be specified then.

I think the easiest way to be compatible with upstream is to create a separate postgresql extension in its own folder.